src/CDI/cores/MC68HC05/M68000Interface.hpp

```diff
@@ -47,8 +47,8 @@ class M68000Interface
     uint8_t modeHost;
 
     void ClearChannels();
-    bool ChannelEnabledMCU(size_t channel);
-    bool ChannelEnabledHost(size_t channel);
+    bool ChannelEnabledMCU(std::size_t channel);
+    bool ChannelEnabledHost(std::size_t channel);
```

Comment:
I'd prefer to let it as it was before and instead add `#include <cstddef>` between `<array>` and `<cstdint>`